canonical-websites/www.ubuntu.com
# Update power page

## Done

Update POWER page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/power>
- Make sure page matches [the copy doc](https://docs.google.com/document/d/1e-vav_b54KlKsi92w3rDu9J14O0gpp8yAanEAIImlE8/edit#)
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/server/power8>
- Make sure the site redirects to <http://0.0.0.0:8001/download/server/power>


## Issue / Card

Fixes #2868
